src/include/cell.i

```diff
@@ -916,6 +937,7 @@ __wt_cell_unpack_dsk(
         unpack->start_txn = WT_TXN_NONE;
         unpack->stop_ts = WT_TS_MAX;
         unpack->stop_txn = WT_TXN_MAX;
+        unpack->oldest_durable_ts = WT_TS_NONE;
         unpack->newest_durable_ts = WT_TS_NONE;
```

Comment:
"newest_durable_ts" to be initialized to WT_TS_MAX.

Reply:
Fixed.